src/components/TreebankService/TreebankService.js

```diff
@@ -67,7 +67,6 @@ class TreebankService extends Component {
           responseFn(error(request, `Unsupported request: ${name}`, ResponseMessage.errorCodes.UNKNOWN_REQUEST));
           break;
         case 'refreshView':
-          responseFn(error(request, `Unsupported request: ${name}`, ResponseMessage.errorCodes.UNKNOWN_REQUEST));
```

Comment:
My understanding of this API call was that sometimes the old Arethusa widget would break and `refreshView` would force it to refresh. I don't think this is a problem with `treebank-react`, so this API call should do nothing.